mitosheet/src/components/elements/MitoProUpgradePrompt.tsx

```diff
@@ -2,23 +2,25 @@ import React from 'react';
 import Row from '../layout/Row';
 import TextButton from './TextButton';
 
-const MitoProUpgradePrompt = (props: {
+const MitoUpgradePrompt = (props: {
     message: string | undefined;
+    proOrEnterprise: 'Pro' | 'Enterprise'
 }): JSX.Element => {
+
     return (
         <div>
             <Row justify='space-between' align='center'>
                 <p className='text-body-1'>
-                    {props.message || 'This is a Mito Pro feature. To access all Mito Pro functionality, please upgrade.'}
+                    {props.message || `This is a Mito ${props.proOrEnterprise} feature. To access all Mito Pro functionality, please upgrade.`}
```

Comment:
If props.proOrEnterprise is Enterprise, then we should update the message to "To access all Mito Enterprise functionality, please upgrade"